core/src/test/scala/unit/kafka/controller/ControllerIntegrationTest.scala

```diff
@@ -100,13 +100,16 @@ class ControllerIntegrationTest extends ZooKeeperTestHarness {
         dataPlaneMetricMap.put(kafkaMetric.metricName().name(), kafkaMetric)
       }
     }
-    assertEquals(1e-0, controlPlaneMetricMap.get("response-total").get.metricValue().asInstanceOf[Double], 0)
+    // we send two update metadata requests on controller startup. One in KafkaController.onControllerFailover,
+    // and one in onReplicasBecomeOffline when partitionsWithoutLeader is empty
+    assertEquals(2e-0, controlPlaneMetricMap.get("response-total").get.metricValue().asInstanceOf[Double], 0)
     assertEquals(0e-0, dataPlaneMetricMap.get("response-total").get.metricValue().asInstanceOf[Double], 0)
-    assertEquals(1e-0, controlPlaneMetricMap.get("request-total").get.metricValue().asInstanceOf[Double], 0)
+    assertEquals(2e-0, controlPlaneMetricMap.get("request-total").get.metricValue().asInstanceOf[Double], 0)
     assertEquals(0e-0, dataPlaneMetricMap.get("request-total").get.metricValue().asInstanceOf[Double], 0)
     assertTrue(controlPlaneMetricMap.get("incoming-byte-total").get.metricValue().asInstanceOf[Double] > 1.0)
     assertTrue(dataPlaneMetricMap.get("incoming-byte-total").get.metricValue().asInstanceOf[Double] == 0.0)
-    assertTrue(controlPlaneMetricMap.get("network-io-total").get.metricValue().asInstanceOf[Double] == 2.0)
+    println(controlPlaneMetricMap.get("network-io-total").get.metricValue().asInstanceOf[Double])
```

Comment:
I guess this can be removed?

Reply:
Changes to this class not required now. reverted the changes